ui/src/components/Actuator.vue

```diff
@@ -0,0 +1,139 @@
+<script setup lang="ts">
+import { getActuator, getMetricsAll } from "@/api/api";
+import { ref, watch } from "vue";
+import ActuatorItem from "./ActuatorItem.vue";
+import SearchBar from "@/components/SearchBar.vue";
+
+const actuator = ref({});
+const metrics = ref({});
+
+const loadActuator = async () => {
+  let result = (await getActuator())["_links"];
+  let list = [];
+  for (let key in result) {
```

Comment:
Not entirely sure what you're trying to do here. Feels duplicate somehow. You're trying to add the key as a value to the item? Why do we need this? Can't we solve this when we're processing the data?

Reply:
Adding the key into the item